### PR TITLE
Tw admin edit project

### PR DESCRIPTION
**Pull request by TCW417**

### CODE REVIEW

File: civictechprojects/forms.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def create_project(request):
     def delete_project(request, project_id):
         project = Project.objects.get(id=project_id)
 
-        if not request.user.username == project.project_creator.username:
+        if not (request.user.username == project.project_creator.username or request.user.is_staff):
```
Comment (marlonkeating) [inline civictechprojects/forms.py]: Now that this logic is becoming more complex, I'd recommend a couple things.

- Put this owner/staff check in a function
- Use parentheses to separate the logical clauses, e.g. ((x == y) or z)

File: civictechprojects/templates/new_index.html [hunk 2]
```diff
@@ -39,6 +39,7 @@
           emailVerified: {{emailVerified|yesno:"true,false"}},
           firstName: '{{firstName}}',
           lastName: '{{lastName}}',
+          isStaff: '{{isStaff}}' === "True" ? true : false
```
Comment (marlonkeating) [inline civictechprojects/templates/new_index.html]: emailVerified is the same sort of value, so let's treat the two consistently, either using emailVerified's 'yesno' construction, or changing it to match isStaff's ternary construction.

File: common/components/common/projects/ContactProjectButton.jsx [hunk 3]
```diff
@@ -150,6 +116,49 @@ class ContactProjectButton extends React.PureComponent<Props, State> {
       </Button>
     );
   }
+
+  displayEditProjectButton() {
```
Comment (marlonkeating) [inline common/components/common/projects/ContactProjectButton.jsx]:
```suggestion
  displayEditProjectButton(): React$Node {
```

File: common/components/common/projects/ContactProjectButton.jsx [hunk 4]
```diff
@@ -150,6 +116,49 @@ class ContactProjectButton extends React.PureComponent<Props, State> {
...
+    if (CurrentUser.userID() === this.props.project.project_creator || CurrentUser.isStaff()) {
+      return (
+      <div>
+        {this._renderEditProjectButton()}
+      </div>)
```
Comment (marlonkeating) [inline common/components/common/projects/ContactProjectButton.jsx]:
```suggestion
      </div>
      );
```

File: common/components/utils/CurrentUser.js [hunk 5]
```diff
@@ -21,6 +21,10 @@ class CurrentUser {
   static lastName(): string {
     return window.DLAB_GLOBAL_CONTEXT.lastName;
   }
+
+  static isStaff() : string {
```
Comment (marlonkeating) [inline common/components/utils/CurrentUser.js]:
```suggestion
  static isStaff() : boolean {
```

**Comment by mmesson** [comment]
is there any other change we need to do to get this merged?

File: civictechprojects/forms.py (hunk 1 above)
Comment (TCW417) [reply civictechprojects/forms.py]: Moved logic to a function in form_helpers.py.

File: common/components/common/projects/ContactProjectButton.jsx (hunk 4 above)
Comment (TCW417) [reply common/components/common/projects/ContactProjectButton.jsx]: Presumably Manuel made this change...

**Comment by TCW417** [comment]
Marlon approved our work just now.

**Pull request merged**